Informer/Controller/EmployeesController.cs

```diff
@@ -20,7 +20,7 @@ public IActionResult Get()
         var employees = _employeeBLL.GetEmployees();
 
 
-        employees.Select(x => new { x.FirstName, Tel = x.PhoneNumber }).ToList();
+        //employees.Select(x => new { x.FirstName, Tel = x.PhoneNumber }).ToList();
```

Comment:
this change is probably unrelated to the goal of this commit and this branch. Why did we need this in the first place? If we do need it, why are you commenting it out? remove it if it's not useful